### DISCUSSION TITLE
Search with Google => opens Safari, even default browser is set to Arc

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [ViennaRSS/vienna-rss#1845] [Q&A]
The latest Vienna always opens Safari and not the system-set default browser.

Is there any trick to change that?

**Comment by barijaona** [comment]
You mean the "Search with Google" (or search with any other search engine which has been defined in Safari's settings…) in web views contextual menu ?

I am afraid there is nothing we can do to change this: it is directly handled by system's WkWebView

**Comment by barijaona** [comment]
Related : https://github.com/Lord-Kamina/SwiftDefaultApps/issues/83

**Comment by Eitot** [comment]
The "Search with \<search provider\>" menu item cannot be modified. However, you can create your own equivalent by following the instructions here: https://superuser.com/a/369969. A menu item will then be added to the menu when text is selected.
